lib/Runtime/Library/IntlExtensionObjectBuiltIns.h

```diff
@@ -0,0 +1,32 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// INTL_ENTRY is a macro that should be defined before this file is included
```

Comment:
Probably would have a compile error anyway but to be extra explicit you could do

```
#ifndef INTL_ENTRY
#error Must define INTL_ENTRY before #including this file.
#endif
```

Reply:
I'll put this in my next PR so as to not have to re-trigger CI (I can also ensure that INTL_OBJECT_ENABLED (or whatever the define is) is also there)